src/main/pages/che-7/overview/ref_hosted_che_faq.adoc

```diff
@@ -0,0 +1,4 @@
+[id="hosted_che_faq_{context}"]
```

Comment:
Given the contents, this really doesn't have to exist as a module. Just add it as an 'additional resource' link.

Reply:
OK done.